modin/experimental/cloud/ray-autoscaler.yml

```diff
@@ -73,6 +73,7 @@ head_node:
         - DeviceName: /dev/sda1
           Ebs:
               VolumeSize: 40
+    SecurityGroupIds: [sg-0790d73053117d1b0]
```

Comment:
For the proxy to work correctly during development. Need to delete before merge.